packages/components/breadcrumbs/src/breadcrumbs.ts

```diff
@@ -126,30 +127,37 @@ export class Breadcrumbs extends ScopedElementsMixin(LitElement) {
       ${this.noHome
         ? nothing
         : html`
-            <a href=${this.homeUrl}><sl-icon name="house"></sl-icon>${isMobile() ? '' : msg('Home')}</a>
-            <sl-icon name="chevron-right"></sl-icon>
+            <a href=${this.homeUrl}><sl-icon name="home-blank"></sl-icon>${isMobile() ? '' : msg('Home')}</a>
+            <sl-icon name="slash-forward"></sl-icon>
           `}
       ${this.breadcrumbs.length > this.collapseThreshold
         ? html`
-            <sl-menu-button fill="link">
-              <sl-icon name="ellipsis" slot="button"></sl-icon>
+            <sl-button fill="link" id="button" @click=${this.#onClick} icon-only aria-label=${msg('More breadcrumbs')}>
+              <sl-icon name="ellipsis"></sl-icon>
+            </sl-button>
+            <sl-popover anchor="button">
               ${this.breadcrumbs
                 .slice(0, -this.collapseThreshold)
-                .map(
-                  ({ element, label }) => html`<sl-menu-item @click=${() => element.click()}>${label}</sl-menu-item>`
-                )}
-            </sl-menu-button>
+                .map(({ url, label }) => (url ? html`<a href=${url}>${label}</a>` : html`${label}`))}
+            </sl-popover>
           `
         : nothing}
       <slot @slotchange=${this.#onSlotchange}></slot>
     `;
   }
 
+  #onClick = (): void => {
+    this.renderRoot.querySelector<Popover>('sl-popover')?.togglePopover();
```

Comment:
The casting top `<Popover>` should not be necessary anymore with the new `HTMLElementTagNameMap` at the top of `popover.ts`.